electron/settings.html

```diff
@@ -473,6 +470,41 @@ <h5>What do these sorting methods mean?</h5>
 
         </div>
       </div>
+      <div id="wlc-pane" value="wlc" rv-showsettingspane="settingsPaneIndex">
+        <div class="settings-container">
+          <div class="settings-row">
+            <h3>All-time Win/Loss Counters</h3>
+            <p>
+              All-time counters keep a running tally of all your wins and losses and will be stored on exit.<br/>
+              The total counter is a count of all your wins and losses across all decks.</p>
+          </div>
+
+          <div class="settings-row">
+            <input rv-checked="showTotalWinLossCounter" class="tf-toggle" type="checkbox" key="showTotalWinLossCounter" data-toggle="toggle"><span class="counter-toggle-span">Show all-time total win/loss counter</span>
+          </div>
+
+          <div class="settings-row">
+            <input rv-checked="showDeckWinLossCounter" class="tf-toggle" type="checkbox" key="showDeckWinLossCounter" data-toggle="toggle"><span class="counter-toggle-span">Show all-time per-deck win/loss counter</span>
+          </div>
+
+          <div class="settings-row reset-all">
+            <button id="all-reset-button" class="btn btn-danger reset-button reset-all-btn" data-deckid="all">Reset Every All-time Counter</button>
```

Comment:
From the UI, it's not clear to me what the difference between these three buttons is. I'll see if I have some suggestions on nomenclature once I've read the code and figured it out...

Edit: I think I see the issue, I haven't actually played any games with a deck, so I don't have any individual deck totals yet. This is probably fine, will come back later if I still have issues with this.

Reply:
Yeah, it makes more sense when you have decks.

Reply:
Yep, looks good after I've messed around with it a bit